### ISSUE TITLE
Re-downloading a previously deleted video doesn't work

### ISSUE THREAD

**Issue by AtlantisTheLostEmpire** [JunkFood02/Seal#1369]
### Checklist

- [X] I'm reporting a bug unrelated to a specific site.
- [X] I've verified that I'm running the [**latest version**](https://github.com/yt-dlp/yt-dlp/releases/latest) of yt-dlp.
- [X] I've verified that I'm running the latest [**stable version**](https://github.com/JunkFood02/Seal/releases/latest/) of Seal or any later [**preview versions**](https://github.com/JunkFood02/Seal/releases).
- [X] I've read the [**Contributing guidelines**](https://github.com/JunkFood02/Seal/blob/main/CONTRIBUTING.md) and [**Code Of Conduct.**](https://github.com/JunkFood02/Seal/blob/main/CODE_OF_CONDUCT.md)
- [X] I've checked that the site i'm trying to download from is in the [**Supported Sites**](https://github.com/yt-dlp/yt-dlp/blob/master/supportedsites.md) list from yt-dlp
- [X] I understand that the issue will be (ignored/closed) if I intentionally remove or skip any mandatory field.

### Describe the bug

I downloaded a video from Instagram using Seal, but I mistakenly downloaded the wrong one and so I deleted it. Later on while browsing the same user's Instagram, I came across the video again and ended up liking it, so I decided to re-download it. Although it appears to successfully download in the app, the video doesn't seem to appear anywhere on my phone.

I've looked for the video in Seal's download list, Google Photos, and extensively in Samsung's "My Files" directory app, but there's nothing. And yes I have new files sorted at the top by date.

So, I decided to do some tests...

I downloaded a different video from the same user's Instagram to see if downloading still worked and it did. I then downloaded a video from a different site (TikTok) and it also worked, so I then deleted it and re-downloaded it again only to get the same problem.

I've rebooted my phone, cleared my app caches for Seal, Google Photos and My Files, and updated my phone OS and apps:
Android... v14
One UI..... v6.1
Seal......... v1.11.3

Side note, I'm well aware Google Photos has this thing where if a picture/video you've deleted is still in the Recycling Bin, downloading it again will cause the new copy not to show up in your gallery until the original has been deleted from the Bin, probably because they share the exact same filename so it gets confused. After deleting each video however, I did empty the Recycling Bin straight after each time because I'm a neat freak.

Additionally, I did another test with a video I downloaded from YouTube, where I deleted it from the My Files directory instead, then deleted it from the My Files Recycling Bin (instead of the Google Photos one). However, re-downloading the video still causes the same issue.

### To Reproduce

1. Download a video from any site using Seal.
2. Delete the video, then delete it from your recycling bin.
3. Re-download the video.
4. Look for the video on your phone, you won't find it.
5. Cry.

### Error reports

There are none.

**Comment by JunkFood02** [comment]
Did you enable Download archive? If so, please remove the item in it or disable it

**Comment by AtlantisTheLostEmpire** [comment]
Yeah it was enabled so I disabled it. I then re-downloaded all the videos I had problems with once again, and they now ALL show up in my gallery. It's a Christmas miracle!

Now I feel like an idiot, but thanks so much JF02!

**Comment by JunkFood02** [comment]
lmao this bot
